flink-connectors/flink-sql-connector-hive-2.3.9/pom.xml

```diff
@@ -115,6 +115,10 @@ under the License.
 									<pattern>com.google</pattern>
 									<shadedPattern>org.apache.flink.hive.shaded.com.google</shadedPattern>
 								</relocation>
+								<relocation>
+									<pattern>org.apache.commons.lang3</pattern>
+									<shadedPattern>org.apache.flink.hive.shaded.org.apache.commons.lang3</shadedPattern>
```

Comment:
What about relocating the entire `org.apache.commons` package? I listed the entire dependency tree and found some other Apache Commons libs, not just common3.

Reply:
I'm not intended to do so. For the case that the hive depends some class like `org.apache.common.xxx` but it hasn't shiped the class `org.apache.common.xxx`. It's  provided by hadoop-lib. 
Without shadding, when we put hadoop-lib, hive-connector together, it works.
But after shade, it won't works as the class `org.apache.common.xxx` has been shaded to `org.flink.shade.xxxx`, and hadoop-lib can't provide it now.